lib/Cake/View/Helper.php

```diff
@@ -546,7 +546,7 @@ protected function _formatAttribute($key, $value, $escape = true) {
  *
  * @param string $message Message to be displayed
  * @param string $okCode Code to be executed after user chose 'OK'
- * @param string $cancelCode Code to be executed after user chose 'Cancel'
+ * @param string $cancelCode Code to be executed after user chose 'Cancel', also executed when okCode doesn't return
```

Comment:
@markstory Do you want another PR with the bug fix for Helper::_confirm for CakePHP 2.9?